### ISSUE TITLE
Simplify and document release process

### ISSUE THREAD

**Issue by barrymcgee** [ubuntudesign/docs.vanillaframework.io#65]
We need a way to easily kick off a docs release when we update Vanilla to ensure they are in sync.

**Comment by nottrobin** [comment]
Discussed with @barrymcgee.

Currently, the hosting of docs.vanillaframework.io is somewhat hacky, and so setting up any automation around it will likely be redundant before too long and there wasted work.

Once https://github.com/ubuntudesign/docs.ubuntu.com/issues/13 is complete, the hosting will be moved to the docs.ubuntu.com application, and we can start to solve this by making improvements on that application.

Then once https://github.com/ubuntudesign/docs.ubuntu.com/issues/12 is completed, the releasing of new documentation will be controlled completely through pushes to the vanilla-framework repository. However, redirecting to the latest version of the documentation will still require a manual update in the docs.ubuntu.com application.

I have just created this new issue: https://github.com/ubuntudesign/docs.ubuntu.com/issues/45, to automate this process as well.

Once all that is completed, releasing a new version of the vanilla-framework documentation will simply involve creating a new version tag on the vanilla-framework repo (this already happens) and then updating the `docs/versions` folder in the master branch.

**Comment by yaili** [comment]
@anthonydillon is this still blocked? should it be moved to backlog?

**Comment by anthonydillon** [comment]
@yaili yes this is still blocked on @nottrobin WSGI charm which is in progress.

**Comment by kwm14** [comment]
This work will be picked up and complete by @WillMoggridge in this [epic](https://app.zenhub.com/workspace/o/ubuntudesign/vanilla-and-base-squad/issues/120).
